src/components/shared/Entity/DetailsForm/useDataPlaneField.ts

```diff
@@ -58,6 +62,29 @@ export default function useDataPlaneField(
 
                 dataPlanesOneOf.push({ const: option, title });
             });
+
+            // TODO (data-planes) - this should not really be needed but a solution to triage the prod issues
+            //  some users are seeing. Need an actual solution.
+            // If we're in edit AND we have a stored data plane then it more than likely came from
+            //  the live_specs_ext view and we should make sure that it is an option
+            if (
+                isEdit &&
+                storedDataPlane &&
+                dataPlanesOneOf.findIndex(
+                    (datum) =>
+                        datum.const.dataPlaneName.whole ===
+                        storedDataPlane.dataPlaneName.whole
+                ) === -1
+            ) {
```

Comment:
Trying to keep this triage scoped heavily. If we're editing and have the stored data plane then make sure it is in the list. While testing this I never saw `storedDataPlane` being undefined while loading in edit.